### PR TITLE
NO-ISSUE: Renamed events to a uniform format

### PR DESCRIPTION
**Pull request by arielireni**
# Assisted Pull Request

## Description

Renamed events to a uniform format by the structure `<event-context>_<past-verb>`.

Signed-off-by: arielireni <aireni@redhat.com>

## List all the issues related to this PR

- [ ] New Feature
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @
/cc @

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: docs/events.yaml [hunk 1]
```diff
@@ -230,7 +230,7 @@ x-events:
     infra_env_id: UUID
     host_name: string
 
-- name: host_set_bootstrap
+- name: host_bootstrap_setted
```
Comment (avishayt) [inline docs/events.yaml:233]:
```suggestion
- name: host_bootstrap_set
```

File: docs/events.yaml [hunk 2]
```diff
@@ -78,29 +78,29 @@ x-events:
...
+- name: after_inactivity_cluster_deregistered
   message: "Cluster is deregistered due to inactivity"
   event_type: cluster
   severity: "info"
   properties:
     cluster_id: UUID
 
-- name: cluster_installing_finished
+- name: cluster_installation_finished
   message: "Successfully finished installing cluster"
```
Comment (avishayt) [inline docs/events.yaml:89]:
```suggestion
- name: cluster_installation_completed
  message: "Successfully completed installing cluster"
```

File: docs/events.yaml [hunk 3]
```diff
@@ -78,29 +78,29 @@ x-events:
...
   message: "Failed installing cluster. Reason: {failure_reason}"
   event_type: cluster
   severity: "critical"
   properties:
     cluster_id: UUID
     failure_reason: string
 
-- name: cluster_cancel_installation
+- name: cluster_installation_cancelled
   message: "Cancelled cluster installation"
```
Comment (avishayt) [inline docs/events.yaml:104]:
```suggestion
- name: cluster_installation_canceled
  message: "Canceled cluster installation"
```

File: docs/events.yaml [hunk 4]
```diff
@@ -499,7 +499,7 @@ x-events:
   properties:
     cluster_id: UUID
 
-- name: disk_slower_supported_speed
+- name: disk_speed_slower_supported
```
Comment (avishayt) [inline docs/events.yaml:502]:
```suggestion
- name: disk_speed_slower_than_supported
```

File: docs/events.yaml [hunk 5]
```diff
@@ -203,15 +203,15 @@ x-events:
...
   message: "Failed to download image: error fetching from storage backend"
   event_type: infra_env
   severity: "error"
   properties:
     infra_env_id: UUID
 
-- name: host_deregistered_cluster
-  message: "Host {host_name} deregistered"
+- name: host_deregistered
+  message: "Host {host_name} deregistered from cluster"
```
Comment (nmagnezi) [inline docs/events.yaml:214]: didn't we just remove "from cluster" ? 
https://github.com/openshift/assisted-service/commit/cb8576bd58d8e7fa8b45107352e74c0da88aca1b#diff-0aa4b24dc559d5a8af68a084c098a96965b65425c389700c66c4657eb890b38bL214

File: docs/events.yaml [hunk 6]
```diff
@@ -420,14 +420,14 @@ x-events:
   properties:
     cluster_id: UUID
 
-- name: image_deleted_after_ignition_update
+- name: image_deleted_because_ignition_updated
```
Comment (nmagnezi) [inline docs/events.yaml:423]: the actual action here is the image deletion, so shouldn't `deleted` somehow be the last word in this name?
@avishayt WDYT?

**Review by nmagnezi — CHANGES_REQUESTED** [review]

File: docs/events.yaml (hunk 6 above)
Comment (avishayt) [reply docs/events.yaml:423]: Sure.  So how about ignition_updated_therefore_image_deleted?

File: docs/events.yaml (hunk 1 above)
Comment (arielireni) [reply docs/events.yaml]: done

File: docs/events.yaml (hunk 2 above)
Comment (arielireni) [reply docs/events.yaml]: done

File: docs/events.yaml (hunk 3 above)
Comment (arielireni) [reply docs/events.yaml]: done

File: docs/events.yaml (hunk 4 above)
Comment (arielireni) [reply docs/events.yaml]: done

File: docs/events.yaml (hunk 5 above)
Comment (arielireni) [reply docs/events.yaml]: yes, removed

File: docs/events.yaml (hunk 6 above)
Comment (arielireni) [reply docs/events.yaml]: done

**Comment by arielireni** [comment]
@nmagnezi @avishayt  Can you please re-review?

**Review by avishayt — APPROVED** [review]

**Review by nmagnezi — APPROVED** [review]
/lgtm

**Pull request merged**